RoofingCompany.sql

```diff
@@ -786,7 +786,7 @@ GO
 
 CREATE VIEW vPartsView
 AS
-SELECT Part.PartName as [Nazwa części], 
+SELECT TOP 100 PERCENT Part.PartName as [Nazwa części], 
```

Comment:
jak jeszcze raz zobaczę te `100 PERCENT` to nikt nie zaliczy :)

Reply:
poprawione